frontend/packages/console-app/src/components/cloud-shell/CloudShellTerminal.tsx

```diff
@@ -33,6 +41,41 @@ const resource = {
 
 const CloudShellTerminal: React.FC<CloudShellTerminalProps> = ({ username, onCancel }) => {
   const [data, loaded, loadError] = useK8sWatchResource<CloudShellResource>(resource);
+  const [workSpacePod, setWorkspacePod] = React.useState<InitResponseObject>(null);
+  const [namespace, setNamespace] = React.useState<string>(null);
```

Comment:
`namespace` state isn't needed. Add namespace to your `InitResponseObject`

Reply:
moved to initObject